src/utils/attribute.js

```diff
@@ -94,6 +94,7 @@ export default class Attribute {
   // Accepts: Day object
   // Returns: First attribute date info that occurs on given day.
   intersectsDay(day) {
+    // console.count('attribute intersectsDay');
```

Comment:
The best opportunity for optimisation IMO is in `refreshDisabledDays()`;

https://github.com/nathanreyes/v-calendar/blob/4474e3676fca1e9cba565086d1bbd01b7bc433de/src/components/Calendar.vue#L619-L623

It calls `intersectsDay()` for each visible date. Which compares each visible day with each disabled date.

The nested loop is `O(N*M)` in time complexity where N is the number of visible dates, and M is the number of disabled dates; Quadratic time complexity❗️

For a three column calendar (~90 days) with 100 disabled dates, that's ~1000 comparisons.

To optimize this to linear time complexity;

1. Create a map of date status (available or disabled), keyed by visible date.
2. Initialize the map by iterating through each visible date and marking it as available
3. Once the map is initialized with each date marked as available, iterate through the disabled dates and mark each specified date as disabled.

The time complexity will be linear `O(N+M)` because the loops are now sequential, not nested, so the duration will be relative to the number of disabled dates plus the number of visible dates.

For a three column calendar with 100 disabled dates, that's ~200 comparisons.